src/Components/Facility/DischargeModal.tsx

```diff
@@ -95,6 +99,15 @@ const DischargeModal = ({
     setFacility(referred_to);
   }, [referred_to]);
 
+  const { data } = useQuery(routes.getConsultation, {
+    pathParams: {
+      id: consultationData.id,
+    },
+  });
+
+  const ConsultationDiagnosisList: ConsultationDiagnosis[] =
```

Comment:
Ensure proper naming conventions for variables. And rely on type inference as useQuery is typesafe.

Ref: https://www.w3schools.com/js/js_conventions.asp#:~:text=Variable%20and%20function%20names%20written%20as%20camelCase

```suggestion
  const diagnoses =
```